example/src/services/BLEService/BLEService.ts

```diff
@@ -77,6 +77,16 @@ class BLEServiceInstance {
       })
   }
 
+  disconnectDeviceById = (id: string) =>
```

Comment:
It's possible to type this is `DeviceId` or even better as `Device['id']`.

Reply:
Done